modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -351,7 +367,11 @@ var CBIWifiFrequencyValue = form.Value.extend({
 				],
 				'ac': [
 					'11a', '5 GHz', true
-				]
+				],
+				'ax': [
+					'11g', '2.4 GHz', this.channels['11g'].length > 3,
+					'11a', '5 GHz', this.channels['11a'].length > 3
+				],
```

Comment:
The extra comma at the end looks unnecessary, as there are no further items. 
`],`